### ISSUE TITLE
Installations: Use level 4 heading for note box

### ISSUE THREAD

**Issue by MaoShizhong** [TheOdinProject/curriculum#28370]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

In our [Installations lesson](https://www.theodinproject.com/lessons/foundations-installations), the note box headed "Copy and paste keyboard shortcuts" uses a level 3 heading instead of a level 4 heading. For consistency and some CSS interactions, note box headings need to be level 4, not level 3.

## Acceptance criteria

Please comment below if you'd like to be assigned to this issue. Please do not open a PR unless you have been assigned by a maintainer.

- [ ] Change the heading level for "Copy and paste keyboard shortcuts" from level 3 to level 4 by using 4 `#` instead of 3 (line 406) in the lesson file.

### Path

Foundations

### Lesson Url

https://www.theodinproject.com/lessons/foundations-installations

**Comment by Sumyth2107** [comment]
I'd like to contribute to this issue.

**Comment by MaoShizhong** [comment]
All yours @Sumyth2107. Let me know if you run into any issues.

**Comment by MaoShizhong** [comment]
@Kaneki1221 You haven't done the PR correctly.
What you've done is create a PR to your own fork's `main` branch, which doesn't have anything to do with TOP's `main` branch here.

We have a [contributing guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md#setting-up-your-local-clone) that contains instructions for how to open a PR from a branch in your fork to TOP's `main`, where it'll have to be reviewed by a maintainer for approval and merge.

**Comment by Kaneki1221** [comment]
Hi @MaoShizhong  could you check the above pull request and let me know if its fixed .
